corehq/apps/domain/views.py

```diff
@@ -538,6 +539,28 @@ def post(self, request, *args, **kwargs):
 
 @require_POST
 @require_can_edit_web_users
+def cancel_repeat_record(request, domain):
+    record = RepeatRecord.get(request.POST.get('record_id'))
+    record.cancel()
+    record.save()
+    return json_response({
+        'success': record.cancelled,
```

Comment:
what if you returned a non-`200` error if this was false instead? that way in your javascript, the `error` method will handle all the failure cases.